src/edit/selection.js

```diff
@@ -116,7 +116,7 @@ class SelectionState {
       this.pm.content.classList.add("ProseMirror-nodeselection")
       this.lastNode = dom
     }
-    let range = document.createRange(), sel = window.getSelection()
+    let range = document.createRange(), sel = this.pm.root.getSelection()
```

Comment:
Can this code be executed when `this.pm.root` is `null`? If so a null check may be worthwhile.